pkg/security/security_profile/profile/selector/selector_proto_dec_v1.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+// Package selector holds profile related files
+package selector
+
+import (
+	proto "github.com/DataDog/agent-payload/v5/cws/dumpsv1"
+
+	cgroupModel "github.com/DataDog/datadog-agent/pkg/security/resolvers/cgroup/model"
+)
+
+// ProtoToWorkloadSelector decodes a Selector structure
+func ProtoToWorkloadSelector(selector *proto.ProfileSelector) cgroupModel.WorkloadSelector {
+	if selector == nil {
+		return cgroupModel.WorkloadSelector{}
+	}
+
+	return cgroupModel.WorkloadSelector{
+		Image: selector.GetImageName(),
+		Tag:   selector.GetImageTag(),
+	}
+}
```

Comment:
Same as encoder func: I wonder if we should rather put those functions in `resolvers/cgroup/` dir  (where the workload selector stands)?